src/mixins/raster-layer.js

```diff
@@ -371,7 +373,17 @@ export default function rasterLayer(layerType) {
   }
 
   function renderPopupHTML(data, columnOrder, columnMap, formatMeasureValue) {
-    let html = ""
+    let html =
+      '<div class="' +
+      _popup_item_copy_class +
+      '">' +
+      '<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">' +
+      '<path d="M10.6668 0.666748H2.66683C1.9335 0.666748 1.3335 1.26675 1.3335 2.00008V11.3334H2.66683V2.00008H10.6668V0.666748ZM10.0002 3.33341L14.0002 7.33341V14.0001C14.0002 14.7334 13.4002 15.3334 12.6668 15.3334H5.32683C4.5935 15.3334 4.00016 14.7334 4.00016 14.0001L4.00683 4.66675C4.00683 3.93341 4.60016 3.33341 5.3335 3.33341H10.0002ZM9.3335 8.00008H13.0002L9.3335 4.33341V8.00008Z" fill="#0089D1"/>' +
```

Comment:
We should remove the hardcoded `fill="#0089D1"` here and set it in css so we can override it, for example with user override in servers.json ($mdc-theme-primary) and with hover state. One trick is to add `fill="currentColor"` on the svg so you can override it with css with `.popup-item-copy {color: $mdc-theme-primary}` from Immerse.

Reply:
Removed the fill from svg and added the css in immerse.